boilerplate/flyte/end2end/end2end.sh

```diff
@@ -18,14 +18,14 @@ pushd ${OUT}
 if [ ! -z "$IMAGE" ]; 
 then
   kind load docker-image ${IMAGE}
-  if [ ${IMAGE_NAME} -eq "flytepropeller" ]
+  if [ "${IMAGE_NAME}" == "flytepropeller" ]
   then
-    sed -i.bak -e "s_${IMAGE_NAME}:.*_${IMAGE}_g" ${OUT}/kustomize/base/propeller/deployment.yaml
+    sed -i.bak -e "s_${IMAGE_NAME}:.*_${IMAGE}_g" "${OUT}"/kustomize/base/propeller/deployment.yaml
```

Comment:
should we continue to use kustomize?

Reply:
Ideally not... might as well change it now I guess...

Reply:
I'll leave that for another PR 😬 ... I want to get this in for 18.1